myuw/templates/base.html

```diff
@@ -152,54 +141,61 @@ <h2 class="sr-only" id="app_messages_desktop_label">MyUW Announcements</h2>
-                         {% endif %}
-                        {% endif %}
+                            {% if user.affiliations.student %}
+                            <li><a {% if is_myuw_academics_page %}class="active"{% endif %} href="/academics/"><i class="fa fa-graduation-cap" aria-hidden="true"></i>Academics</a></li>
+                            <!--<li><a href="/huskyexp/"><i class="fa fa-paw" aria-hidden="true"></i>Husky Experience</a></li>-->
+                            {% endif %}
 
-                        <li><a {% if is_myuw_academic_calendar_page %}class="active"{% endif %} href="/academic_calendar/"><i class="fa fa-calendar-check-o" aria-hidden="true"></i>Calendar</a></li>
-                        <li><a {% if is_myuw_accounts_page  %}class="active"{% endif %} href="/accounts/"><i class="fa fa-credit-card" aria-hidden="true"></i>Accounts</a></li>
-                        {% if user.affiliations.student %}
-                        <li><a {% if is_myuw_notices_page %}class="active"{% endif %}  href="/notices/"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i>Notices</a></li>
-                        {% endif %}
-                        <li><a {% if is_myuw_profile_page %}class="active"{% endif %} href="/profile/"><i class="icon-user icon-user-female" aria-hidden="true" style="font-size:14px;"></i>Profile</a></li>
+                            {% if instructor_schedule_enabled %}
+                              {% if user.affiliations.instructor %}
+                            <li><a {% if is_myuw_teaching_page %}class="active"{% endif %} href="/teaching/"><i class="fa fa-pencil-square-o" aria-hidden="true"></i>Teaching</a></li>
+                             {% endif %}
+                            {% endif %}
 
-                    </ul>
+                            <li><a {% if is_myuw_academic_calendar_page %}class="active"{% endif %} href="/academic_calendar/"><i class="fa fa-calendar-check-o" aria-hidden="true"></i>Calendar</a></li>
+                            <li><a {% if is_myuw_accounts_page  %}class="active"{% endif %} href="/accounts/"><i class="fa fa-credit-card" aria-hidden="true"></i>Accounts</a></li>
+                            {% if user.affiliations.student %}
+                            <li><a {% if is_myuw_notices_page %}class="active"{% endif %}  href="/notices/"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i>Notices</a></li>
+                            {% endif %}
+                            <li><a {% if is_myuw_profile_page %}class="active"{% endif %} href="/profile/"><i class="icon-user icon-user-female" aria-hidden="true" style="font-size:14px;"></i>Profile</a></li>
+
+                        </ul>
 
+                    </div>
+                    {% endblock %}
+                    <!-- end app_navigation -->
+
+                    <!-- Unless applicant -->
```

Comment:
@ezturner Here is where we want to wrap an unless conditional around the welcome block.